CHANGELOG.md

```diff
@@ -4,6 +4,30 @@
 
 ### Enhancements
 
+* Update bugsnag-cocoa to v6.9.6:
+	
+	* Improve accuracy of app hang event information to better reflect state at time of detection.
```

Comment:
The indentation looks a bit odd - I think because of tabs instead of spaces...

Reply:
fixed